docs/userguide.rst

```diff
@@ -93,6 +93,8 @@ For detailed information, see the API reference of the associated class.
 +--------------+-----------------------------------------------------------------------------+-------------------------------------+----------------------------------------------------+
 | file         | ``file:/tmp/file-to-push``                                                  | n/a                                 | Obtain a single file-backed item of various types. |
 +--------------+-----------------------------------------------------------------------------+                                     |                                                    |
+| cgw          | ``cgw:/tmp/yaml-file-to-push``                                               |                                     |                                                    |
```

Comment:
I think there is still a whitespace error here.  This time in the third and later instances of `|` character.  It comes because my suggestion added one space near the beginning and didn't delete a space from later in the line.